acme/http_challenge.go

```diff
@@ -4,56 +4,43 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
-	"net"
 	"net/http"
-	"strings"
 	"time"
 )
 
+type httpChallengeMethod interface {
```

Comment:
The commit https://github.com/xenolf/lego/commit/61a0912a11df96a7956b60a5a1b66c63f29f402d fixes that